docs/getting_started/install_manually.md

```diff
@@ -212,290 +212,116 @@ Open your project in the browser and you should see the welcoming page.
+    After installation you can replace `php -d memory_limit=-1 composer.phar` with `composer`.
 
-``` bash
-sudo vi /private/etc/apache2/users/ez1.lh.conf
-```
+### Run installation scripts
 
-e. Modify virtual host file **vhost.template.**
+Composer will look inside the `composer.json` file and install all of the packages required to run eZ Platform.
+The `bin/console` script will then install eZ Platform.
 
-Replace the `---USER_ID---` variable (used in lines 10 and 17) with your current user ID. Use `whoami` command to get effective user ID of the currently logged user. If you want to use the default virtual host template (delivered with eZ Platform package) all you have to do is set up lines 7, 8, 9, 10, 17, 25 and 33:
+#### Install dependencies with Composer
 
-f. Restart Apache 2 server:
+From the folder into which you downloaded eZ Platform, run:
 
 ``` bash
-sudo apachectl restart
+php -d memory_limit=-1 composer.phar install
 ```
 
-g. Install required dependencies using Composer:
-
-``` bash
-composer install
-```
+Once the installer gets to the point that it creates `app/config/parameters.yml`, you will be presented with a few decisions:
 
-When Composer asks you for the token you must log in to your GitHub account and edit your profile. Go to the Personal access tokens link and Generate new token with default settings. Be aware that the token will be shown only once, so do not refresh the page until you paste the token into Composer prompt. This operation is performed only once when you install eZ Platform for the first time.
+1. Choose a [secret](http://symfony.com/doc/current/reference/configuration/framework.html#secret); it should be a random string, made up of up to 32 characters, numbers, and symbols. This is used by Symfony when generating [CSRF tokens](http://symfony.com/doc/current/book/forms.html#forms-csrf), [encrypting cookies](http://symfony.com/doc/current/cookbook/security/remember_me.html), and for creating signed URIs when using [ESI (Edge Side Includes)](http://symfony.com/doc/current/book/http_cache.html#edge-side-includes).
+2. You can accept the default options for `database_driver`, `database_host` and `database_port`
+3. For `database_name` and `database_user`, replace them if you customized those values during configuration.
+4. If you set a password for your database user, enter it when prompted for `database_password`.
 
-h. Change directory permissions:
+The installer should continue once you've completed the manual portion of the installation process.
 
-``` bash
-rm -rf var/cache/* var/logs/* var/sessions/*
-sudo chmod +a "_www allow delete,write,append,file_inherit,directory_inherit" var web/var
-sudo chmod +a "`whoami` allow delete,write,append,file_inherit,directory_inherit" var web/var
-```
+#### Install eZ Platform
 
-i. Install eZ Platform:
+Create `clean` installation in development environment and a database with:
```

Comment:
It can suggest that `composer ezplatform-install` will also create a database. It can be mentioned that user needs to create database first.